src/main/server.ts

```diff
@@ -33,26 +36,20 @@ function getSSLOptions (): any {
   }
 }
 
-function forkListenerProcesses (numberOfCores: number) {
-  for (let i = 0; i < numberOfCores; i++) {
-    cluster.fork()
+function applicationPort (): any {
+  let port: any = process.env.PORT
```

Comment:
`process.env.PORT || defaultPort` ? 
Is the log message about using the default port still useful?

Reply:
I think it is.

Reply:
Proved itself at least once today 📦

Reply:
but every time a developer starts up their local?

Reply:
Why is it a problem?